### PR TITLE
Tools: script updated for Ubuntu 16.x

### PR DESCRIPTION
**Pull request by AndKe**
No older versions of Ubuntu is affected by this script.
-solves https://github.com/ArduPilot/ardupilot/issues/3894

### CODE REVIEW

**Comment by lucasdemarchi** [comment]
I would prefer 2 things to be implemented differently:
  - If you only grep the archive, it could be that there's "Ubuntu 16.04" in it as a comment which would be a false trigger to consider it as Ubuntu 16.04. In configure-ci.sh script we get the string only from the correspondent key
  - I think it would be better to keep common packages in a single string. Otherwise we will keep duplicating it as new versions come. Most of the packages have the same names in different versions of the distro

File: Tools/scripts/install-prereqs-ubuntu.sh [hunk 1]
```diff
@@ -14,6 +12,15 @@ BEBOP_PKGS="g++-arm-linux-gnueabihf"
 UBUNTU64_PKGS="libc6:i386 libgcc1:i386 gcc-4.9-base:i386 libstdc++5:i386 libstdc++6:i386"
 ASSUME_YES=false
 
+if grep -q "Ubuntu 16" /etc/os-release
+then
+	# Packages for Ubuntu 16.x
+	SITL_PKGS="g++ python-pip python-matplotlib python-serial python-wxgtk3.0 python-scipy python-opencv python-numpy python-pyparsing ccache realpath"
```
Comment (OXINARF) [inline Tools/scripts/install-prereqs-ubuntu.sh]: I would prefer that we left the equal packages general and then just add the different packages with `+=`

**Comment by AndKe** [comment]
@lucasdemarchi  - I kept it in separate strings, to have it readable if future versions were very different. (after all, we still know people at 14.04)  - but on the other hand, it's only one change now..

and I cannot imagine why /etc/os-release would contain any other major version numbers, but I fixed it so it looks for specific line as you asked :)

**Comment by lucasdemarchi** [comment]
Just installed ubuntu 16.04 in a vm for test:

```
ubuntu@ubuntu-xenial:~/ardupilot$ ./Tools/scripts/install-prereqs-ubuntu.sh 
./Tools/scripts/install-prereqs-ubuntu.sh: line 19:  python-wxgtk3.0: command not found
```

**Comment by AndKe** [comment]
@lucasdemarchi  thanks - that's embarrasing to wake up to :)   I just hammered down those last trivial changes, added too much "space"  :) - fixed now.

**Comment by OXINARF** [comment]
Actually Lucas fixed it on top of your commit and has already pushed to master along with other fixes. Thanks!

**Pull request closed without merging**